src/main/kotlin/app/widgets/recordButton/RecordButton.kt

```diff
@@ -0,0 +1,86 @@
+package app.widgets.recordButton
+
+
+import app.ui.styles.ButtonStyles
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.application.Platform
+import javafx.geometry.Pos
+import javafx.scene.Cursor
+import javafx.scene.layout.VBox
+import javafx.scene.paint.Color
+import tornadofx.*
+
+
+import java.util.*
+import kotlin.concurrent.timerTask
+
+class RecordButton(var onClickCallback: () -> Unit = ::println, var animationCompletedCallback: () -> Unit = ::println, var stopClickedCallback: () -> Unit = ::println) : VBox() {
+
+
+    val circle = RecordingAnimation()
+    val dotsAn = DotsAnimation()
```

Comment:
I'd name this dotsAnim if you really need to abbreviate but dotsAnimation is probably better